### PR TITLE
Added footer to bottom of every page 

### LINKED ISSUE #169: All pages need footers
The footer on the homepage should be on ever page via an include like the navbar.

### PR DESCRIPTION
**Pull request by vkr96**
Footers for all pages like the one on the main page 

fixes #169

### CODE REVIEW

File: client/components/footer/footer.controller.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+angular.module('observatory3App')
+  .controller('MainCtrl', function ($scope, $http) {
```
Comment (agundy) [inline client/components/footer/footer.controller.js]: This shouldn't be called MainCtrl. And doesn't look like a controller is even needed at this point. You could leave it out for now. We can add it back when we need more logic/functionality

**Comment by agundy** [comment]
I also see you adding the footer too a component and adding it to other pages but you left it on the main page. You should remove the footer from the main page and put in the include so that we only have to update the footer in one location.

**Comment by vkr96** [comment]
I'm so sorry for the disjointed commits.....

**Comment by seveibar** [comment]
Looks good to me! Any last thoughts @agundy?

**Comment by kmcnellis** [comment]
The footer is still missing on the user profile, and the signup/login pages, it looks like

**Comment by vkr96** [comment]
That should fix the problem

**Comment by kmcnellis** [comment]
Could you also add a link to the github in the footer?

**Comment by vkr96** [comment]
If you click on issues it leads to the github

**Pull request merged**